### DISCUSSION TITLE
Audiences

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#92] [Controlled Vocabularies]
Which controlled vocabulary should be used in Artsdata for audience types?

**Comment by MaudeFJ** [comment]
Hello, 
Has there been an update about this since last year ?

**Reply by MaudeFJ** [reply]
Bonjour, @saumier @fjjulien 
Je me rends compte que l'audience que je mets dans mon calendrier ne se retrouve pas dans arts data.
Est-ce qu'on pourrait reprendre cette discussion pour qu'il y ait éventuellement un champs audience dans le modèle d'arts data ?
Merci
